src/qt/bitcoingui.cpp

```diff
@@ -251,6 +253,9 @@ BitcoinGUI::~BitcoinGUI()
     delete m_app_nap_inhibitor;
     MacDockIconHandler::cleanup();
 #endif
+#ifdef BITCOIN_QT_WIN_TASKBAR
+    delete m_taskbar_progress;
```

Comment:
Maybe we should keep the current interface, and let QObject parent/child deletion handle this?